recipes/code-aster/conda_build_config.yaml

```diff
@@ -0,0 +1,42 @@
+boost:
```

Comment:
The conda-forge channel has a [repository of package pinnings ](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml) that is loaded into our build environment automatically. You should not be defining your own build variants with a conda_build_config,yaml unless you are adding variants that are not defined in the conda-forge pinnings.

Reply:
Thanks, I did not know about that. I am wondering if this can work as expected : I know that this complex software will only work using a gcc/gfortran 8 version (and not yet on later ones), only with some hdf5 versions... Should I put these constraints in the meta.yaml file ?